### ISSUE TITLE
Provisioning latest energy supply generator data - extra column

### ISSUE THREAD

**Issue by tomalrussell** [nismod/nismod2#167]
Running `provision/install_energy_supply.sh` gives an error when copying data into postgres.
```
./data/energy_supply/database_full/GeneratorData.csv
ERROR:  extra data after last expected column
CONTEXT:  COPY GeneratorData, line 2: "1,1,Aberthaw GT,53,18,,0,51,,,2015,2030,1,4,3,,,18"
```

`GeneratorData.csv` seems to have an extra couple of columns right at the end:

```csv
GenNum,Type,GeneratorName,GasNode,BusNum,EH_Conn_Num,MinPower,MaxPower,PumpStorageCapacity,BatteryStorage,Year,Retire,SysLayer,CoolingWater_Source,Cooling_Method,Inter_conn,,
1,1,Aberthaw GT,53,18,,0,51,,,2015,2030,1,4,3,,,18
```

- the next-to-last seems to be always empty
- the last column has numbers for most rows but no column heading

**Comment by LahiruJayasuriya** [comment]
These columns are redundant. I've released a cleaned version in `energy_supply_data_v0.9.10`

**Comment by tomalrussell** [comment]
Closed by db46ff0
